internal/dag/httpproxy_processor.go

```diff
@@ -643,6 +650,16 @@ func (p *HTTPProxyProcessor) processHTTPProxyTCPProxy(validCond *contour_api_v1.
 		return false
 	}
 
+	lbPolicy := loadBalancerPolicy(tcpproxy.LoadBalancerPolicy)
+	switch lbPolicy {
+	case LoadBalancerPolicyCookie, LoadBalancerPolicyRequestHash:
+		validCond.AddWarningf("SpecError", "IgnoredField",
```

Comment:
With this and other warnings, not sure entirely if this is the right context to put on the warning, I saw there were constants somewhere, could use some context on what would be preferred

Reply:
I think for this one we might want to use `contour_api_v1.ConditionTypeTCPProxyError`, seems more consistent with other errors for TCPProxy. We don't necessarily have a perfectly consistent scheme across all types.